pkg/vz/vm_darwin.go

```diff
@@ -334,6 +335,58 @@ func attachDisks(driver *driver.BaseDriver, vmConfig *vz.VirtualMachineConfigura
 	}
 	configurations = append(configurations, diffDisk)
 
+	if len(driver.Yaml.AdditionalDisks) > 0 {
```

Comment:
This condition is not needed. In case of length 0 for loop should take care i believe